arrow-data-source/pom.xml

```diff
@@ -145,6 +145,30 @@
       <artifactId>scalatest_${scala.binary.version}</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-1.2-api</artifactId>
+      <version>2.17.0</version>
```

Comment:
Change version to 2.17.1

Reply:
Just changed the version to 2.17.1. Thanks!